src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/entity/AdditionalConditionUploadSummary.kt

```diff
@@ -0,0 +1,75 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity
+
+import java.time.LocalDateTime
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.Lob
+import javax.persistence.ManyToOne
+import javax.persistence.Table
+import javax.validation.constraints.NotNull
+
+@Entity
+@Table(name = "additional_condition_upload_summary")
+data class AdditionalConditionUploadSummary(
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @NotNull
+  val id: Long = -1,
+
+  @ManyToOne
+  @JoinColumn(name = "additional_condition_id", nullable = false)
+  val additionalCondition: AdditionalCondition,
```

Comment:
This one doesn't have the licenceId as foreign key - which I think is correct - but not consistent with the UploadDetail

Reply:
Yes - this one does have the relationship managed and is associated with an `additionalCondition` entity.  The uploadDetail is effectively stand-alone, and not managed via any JPA relationships.